packages/wrapped-keys/esbuild.config.js

```diff
@@ -1,7 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+
 const esbuild = require('esbuild');
 
-(async () => {
-  await esbuild.build({
+const ensureDirectoryExistence = (filePath) => {
+  const dirname = path.dirname(filePath);
+  if (fs.existsSync(dirname)) {
+    return true;
+  }
+  ensureDirectoryExistence(dirname);
+  fs.mkdirSync(dirname);
+};
+
+const wrapIIFEInStringPlugin = {
+  name: 'wrap-iife-in-string',
+  setup(build) {
+    build.onEnd((result) => {
+      if (result.errors.length > 0) {
+        console.error('Build failed with errors:', result.errors);
+        return;
+      }
+
+      result.outputFiles.forEach((outputFile) => {
+        let content = outputFile.text;
+
+        // Use JSON.stringify to safely encode the content
+        const wrappedContent = `module.exports = ${JSON.stringify(content)};`;
```

Comment:
Importing a JS file as a string is totally not worth it when you can just wrap it with quotes when building it. Specially on both node.js and browser envs

Reply:
Do you think importing is worth it from the perspective of tree shaking?
Perhaps we use named exports so when the package is consumed unused action can be tree shook by the users bundler.

Reply:
Changed no named exports